### PR TITLE
[Outlook] (preview) Update internetHeaders and officeTheme

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[772e2d8](https://github.com/OfficeDev/office-js-docs-pr/pull/1134/commits/772e2d86e8d6f4cc0b6ae1dd56ed45abf9e48674)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.4/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.5/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.6/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.7/office.context.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C25%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906252106201568-1134%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=8q1vtIcsR%2BfMrJ%2FUejmaqNZr5PSLpNZp17GufAjpowQ%3D&st=2019-06-25T21%3A02%3A37Z&se=2019-07-26T21%3A07%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[003ce5b](https://github.com/OfficeDev/office-js-docs-pr/pull/1134/commits/003ce5b4897d638e441365a0fa183361576c5446)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.4/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.5/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.6/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.7/office.context.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906261514588364-1134%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=boAQ1pFVV80jbg%2FTnCHqDnkej9smFvae0MIJ0SA2XeQ%3D&st=2019-06-26T15%3A11%3A17Z&se=2019-07-27T15%3A16%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6121134](https://github.com/OfficeDev/office-js-docs-pr/pull/1134/commits/61211345634b74135074d697644ad6f20681b8d5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.mailbox.item.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/preview-requirement-set/outlook-requirement-set-preview.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.3/outlook-requirement-set-1.3.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.4/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.4/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.5/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.5/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.6/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.6/office.context.md) | :white_check_mark:Succeeded | |
[docs/reference/objectmodel/requirement-set-1.7/office.context.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-ot-ih/docs/reference/objectmodel/requirement-set-1.7/office.context.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906261516336209-1134%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=g8QIyZzwqhjxtytaqQaVkUTjNI3z6tirGHs8YT%2Bl7Vg%3D&st=2019-06-26T15%3A12%3A41Z&se=2019-07-27T15%3A17%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/objectmodel/preview-requirement-set/office.context.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ function write(message){
 Provides access to the properties for Office theme colors.
 
 > [!NOTE]
-> This member is not supported in Outlook on iOS or Android.
+> This member is only supported in Outlook on Windows.
```
Comment (davidchesnut) [inline docs/reference/objectmodel/preview-requirement-set/office.context.md]: Subtle change, but better readability. Nice!

Comment (ElizabethSamuel-MSFT) [reply docs/reference/objectmodel/preview-requirement-set/office.context.md]: And I think Mac is not supported either so definitely more accurate. Thx!

**Pull request merged**